typescript/packages/plugins/avnu/tsup.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from "tsup";
```

Comment:
Make sure the tsup.config.ts file follows the same structure as other plugins. You can generate all these files using the plugin generator: https://ohmygoat.dev/plugins#using-the-plugin-generator